apps/golang/install

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+VERSION="1.19.2"
```

Comment:
if you change this and the references to lowercase `version` then our updater will be able to update it at some point as well as our metadata parser will be able to find the app version

Reply:
also stable version is now 1.19.3, which is why an updater within pi-apps would be necessary

for personal reference, this will get the current version to add to a pi-apps auto updater script
`curl -s https://go.dev/dl/?mode=json | jq -r '.[0].version'`

Reply:
I made `VERSION` lowercase, and removed `VERSION="1.19.2"`
Is that all that was needed?

Reply:
no, `version=1.19.3` need to exist there... you weren't supposed to remove it, just change from uppercase to lowercase